extensions/amp-instagram/1.0/storybook/Basic.js

```diff
@@ -13,29 +11,25 @@ import {Instagram} from '../component';
 export default {
   title: 'Instagram',
   component: Instagram,
-  decorators: [withKnobs],
+  args: {
+    width: 500,
+    height: 600,
+    shortcode: 'B8QaZW4AQY_',
+    captioned: false,
```

Comment:
This new boolean ui is way nicer than the checkbox :) 
<img width="311" alt="Screen Shot 2021-09-02 at 10 05 15 AM" src="https://user-images.githubusercontent.com/10456171/131858212-43ff0055-612c-44c4-965c-47ebe22f1e14.png">